tests/test_frequenz_dispatch.py

```diff
@@ -534,14 +546,15 @@ async def test_multiple_dispatches_unify_running_intervals(
     service.start()
 
     receiver = await service.new_running_state_event_receiver(
-        "TEST_TYPE", unify_running_intervals=True
+        "TEST_TYPE", merge_strategy=MergeByType()
```

Comment:
So here it would be:
```py
        "TEST_TYPE", merge_strategy=MergeByType(service)
```

If there are cases where you don't have a `Dispatcher` instance, then maybe it could take the dispatches dict directly, although it is more bug prone, if for some reason the object gets replaced or updated inside the `DispatchScheduler`, then the merge strategy will get out of sync, so I'm more inclined to get the dict every time from some other object we need is much less likely to be replaced than a `dict`. Worse case it could get a callable to get the `dict`, although is not super user-friendly, so maybe just change the test to test merge strategies in places where a `Dispatcher` instance is available.